frontend/packages/console-shared/src/components/formik-fields/text-column-field/TextColumnItem.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import { useDrag, useDrop } from 'react-dnd';
+import { useTranslation } from 'react-i18next';
+import {
+  Flex,
+  FlexItem,
+  TextInputTypes,
+  Button,
+  ButtonVariant,
+  ButtonType,
+  Tooltip,
+} from '@patternfly/react-core';
+import { MinusCircleIcon, GripLinesVerticalIcon } from '@patternfly/react-icons';
+import { InputField } from '@console/shared';
+import { TextColumnItemProps, ItemTypes, DragItem } from './text-column-types';
+
+const TextColumnItem: React.FC<TextColumnItemProps> = ({
+  idx,
+  name,
+  tooltip,
+  placeholder,
+  isReadOnly,
+  onChange,
+  rowValues,
+  disableDeleteRow,
+  arrayHelpers,
+  dndEnabled,
+}) => {
+  const { t } = useTranslation();
+  const [, drag, preview] = useDrag({
+    item: { type: ItemTypes.TextColumn, id: `${ItemTypes.TextColumn}-${idx}`, idx },
+  });
+  const [{ opacity }, drop] = useDrop({
+    accept: ItemTypes.TextColumn,
+    collect: (monitor) => ({
+      opacity: monitor.isOver() ? 0 : 1,
+    }),
+    hover(item: DragItem) {
+      if (item.idx === idx) {
+        return;
+      }
+      arrayHelpers.swap(item.idx, idx);
+      if (onChange) {
+        const values = [...rowValues];
+        [values[idx], values[item.idx]] = [values[item.idx], values[idx]];
+        onChange(values);
+      }
+      // monitor item updated here to avoid expensive index searches.
+      item.idx = idx;
+    },
+  });
+
+  return (
+    <div ref={(node) => preview(drop(node))} style={{ opacity }}>
+      <Flex style={{ marginBottom: 'var(--pf-global--spacer--sm)' }}>
+        {!!dndEnabled && (
+          <FlexItem style={{ cursor: 'move' }}>
+            <div ref={drag}>
+              <GripLinesVerticalIcon />
```

Comment:
The icon seems to be different in the designs

<img width="1305" alt="Screenshot 2021-04-20 at 2 00 56 PM" src="https://user-images.githubusercontent.com/9278015/115364280-dc8e6d80-a1e0-11eb-86f4-18fa8e1e3965.png">

Reply:
check this one in the design doc

![image](https://user-images.githubusercontent.com/5129024/115367773-1d3bb600-a1e4-11eb-85d7-f47483a53471.png)

confirmed with design team have to use `GripVertical` cc @divyanshiGupta

Reply:
Yes, the dnd icon will be replaced with pf icon across console so here we need to use the pf one too.